src/store/index.js

```diff
@@ -149,8 +149,8 @@ export default new Vuex.Store({
       commit('addState', payload);
     },
     createAList({ commit }, payload) {
```

Comment:
there are two methods with the same name. Method names should be distinct and have more context.

Reply:
There is no issue to change the name in this situation, yet please could you inform me what rules you would like me to follow?